src/utils/constantValues.ts

```diff
@@ -0,0 +1,58 @@
+export const autoCompleteValuesArray = [
```

Comment:
As I was clicking into this PR, I was wondering if you had done something like this to share values across the `TextInput` and `Select` component. And yes, you did.

I feel this file might be a good place for color groupings that could be use for color assignments across various components.